src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int maxValue = 100;

```

Comment:
1. constants are private static final and placed before other variables
2. NAMING_CONVENTION_FOR_CONSTANT
3. Name of constant should provide as much understanding of what's inside as possible
```suggestion
    private static final int MAX_BALL_NUMBER = 100;
    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();
```